api_server.py

```
@@ -161,7 +161,8 @@ def check_rules(available_ids: List[int]) -> Dict:
     with get_db() as conn:
         placeholders = ",".join("?" * len(available_ids))
         rows = conn.execute(
-            f"SELECT role, skills FROM crew WHERE id IN ({placeholders})", available_ids
+            f"SELECT role, skills FROM crew WHERE id IN ({placeholders})",
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)